src/app/components/client/UpsellDialog.tsx

```diff
@@ -13,10 +13,10 @@ import { TabList } from "./TabList";
 import { Button } from "../client/Button";
 import { useL10n } from "../../hooks/l10n";
 import ModalImage from "../client/assets/premium-upsell-dialog-icon.svg";
-import styles from "./PremiumUpsellDialog.module.scss";
+import styles from "./UpsellDialog.module.scss";
 import { useTelemetry } from "../../hooks/useTelemetry";
 
-export interface PremiumUpsellDialogProps {
+export interface UpsellDialogProps {
```

Comment:
Haha, didn't want to call this `PlusUpsellDialogProps` in case we change the name again? :P

Reply:
That might have been my train of thoughts …

Reply:
… but generally, I just now think that avoiding branding-specific naming within code if reasonable leads to less future inconsistencies and clean-up.